src/streamlink/plugins/eltrecetv.py

```diff
@@ -13,27 +12,31 @@ def can_handle_url(cls, url):
         return cls._url_re.match(url)
 
     def _get_streams(self):
-
-        video_url_found_hls = ""
-
         if "eltrecetv.com.ar/vivo" in self.url.lower():
-            http.headers = {'Referer': self.url,
-            'User-Agent': "ArtearPlayer/3.2.42 (Linux;Android 4.4.2) ExoPlayerLib/1.5.9"}
-            video_url_found_hls = "http://stream.eltrecetv.com.ar/live13/13tv/13tv1/playlist.m3u8"
+            try:
+                http.headers = {'Referer': self.url,
+                'User-Agent': "ArtearPlayer/3.2.42 (Linux;Android 4.4.2) ExoPlayerLib/1.5.9"}
+                res = http.get('https://api.iamat.com/metadata/atcodes/eltrece')
+                live_search = res.text
+                live_search = live_search[live_search.index('"vivo":{"provider":"youtube","youtubeVideo":')+45 :]
+                live_search = live_search[: live_search.index('"')]
+                LIVE_URL_FOUND = 'https://www.youtube.com/watch?v=' + live_search
```

Comment:
Variables in caps should be avoided, the convention is that caps are reserved for constants. I would make the `https://www.youtube.com/watch?v=` string a constant and put it at the class or module level, eg.
```python
class ElTreceTV(Plugin):
    LIVE_URL = "https://www.youtube.com/watch?v="
```
I'd also use `string.format` as that is generally considered superior to string concatenation, (`"https://www.youtube.com/watch?v={0}".format(yt_id)`)